### PR TITLE
adding team: 2024 NASA Openscapes Champions

### PR DESCRIPTION
**Pull request by jules32**
We request adding a new team to the Openscapes JupyterHub for the 2024 NASA Champions Cohort. 

This PR supersedes https://github.com/2i2c-org/infrastructure/pull/3930

### CODE REVIEW

File: config/clusters/openscapes/common.values.yaml [hunk 1]
```diff
@@ -47,17 +47,17 @@ basehub:
...
-            - nasa-openscapes-workshops:ChampionsAccess-2i2c
-            - nasa-openscapes-workshops:LongtermAccess-2i2c
-            - nasa-openscapes-workshops:WorkshopAccess-2i2c
+            - nasa-openscapes-workshops:nasa-champions-2024
+            - nasa-openscapes-workshops:ChampionsAccess-2i2c # legacy, will delete - has only one user from 2023 cohort
+            - nasa-openscapes-workshops:WorkshopAccess-2i2c # From SWOT workshop 2024-02-13
             - nasa-openscapes-workshops:emit-methane-plume-2024-03-14
           kubespawner_override:
-            image: openscapes/python:39dffde
+            image: openscapes/python:4f340eb
```
Comment (yuvipanda) [inline config/clusters/openscapes/common.values.yaml:60]: Is this change intentional?

Comment (jules32) [reply config/clusters/openscapes/common.values.yaml:60]: no, I think this was a mistake. I will fix!
```suggestion
            image: openscapes/python:4f340eb
```

Comment (consideRatio) [reply config/clusters/openscapes/common.values.yaml:60]:
```suggestion
            image: openscapes/python:39dffde
```

**Review by consideRatio — APPROVED** [review]
